### PR TITLE
[ISSUE-105] - improve issue handling

### PR DESCRIPTION
**Pull request by baranowb**
https://github.com/jboss-set/aphrodite/issues/105

### CODE REVIEW

File: src/main/java/org/jboss/set/aphrodite/issue/trackers/common/AbstractIssueTracker.java [hunk 1]
```diff
@@ -140,6 +140,20 @@ public boolean urlExists(URL url) {
         return url.getHost().equals(baseUrl.getHost());
     }
 
+    @Override
+    public String getTrackerID() {
+        return convertToTrackerID(this.baseUrl);
+    }
+
+    public static String convertToTrackerID(URL url) {
```
Comment (baranowb) [inline src/main/java/org/jboss/set/aphrodite/issue/trackers/common/AbstractIssueTracker.java]: and iterate over every tracker service, until it either finds one and perform operation or fail with exception? This does not make much sense.

Comment (elguardian) [inline src/main/java/org/jboss/set/aphrodite/issue/trackers/common/AbstractIssueTracker.java]: I'm not saying you modify this one.... I'm saying both logic should match. So change the urlExists to use this method.

Comment (baranowb) [inline src/main/java/org/jboss/set/aphrodite/issue/trackers/common/AbstractIssueTracker.java]: Now you did :)

**Comment by baranowb** [comment]
@rpelisse - if you are ok with this I will merge it today.

**Comment by rpelisse** [comment]
@baranowb  sorry, on PTO yesterday - no issue on my side with those change.

**Pull request merged**